app/buttle/preparation_functions.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+from app.buttle.preparations_classes import Knight, Weapon, Effect, Potion, \
+    Armour
+
+
+def list_of_knight_instances_from_dict(knights_config: dict) -> list[Knight]:
+    knights_instances = []
+    for knight_name in knights_config:
+        knight_dict = knights_config[knight_name]
+        # armour
```

Comment:
Maybe you need to delete comments here and below according to checklist in README.md

Reply:
Fixed!